src/main/scala/edu/cmu/cs/obsidian/parser/ImportProcessor.scala

```diff
@@ -109,12 +109,11 @@ object ImportProcessor {
 
     /*check this */
     def filterTags(contracts: Seq[Contract]): Seq[Contract] = {
-        contracts.map(c => c match {
```

Comment:
I'll accept this change, but minor changes like this make PRs harder to review and make commit logs harder to understand. It's inconvenient, but it's better form to propose these kinds of changes separately.